python/ray/util/client/logsclient.py

```diff
@@ -57,9 +57,8 @@ def _log_main(self) -> None:
                 logger.info("Server disconnected from logs channel")
             else:
                 # Some other, unhandled, gRPC error
-                logger.error(
+                logger.exception(
                     f"Got Error from logger channel -- shutting down: {e}")
-                raise e
```

Comment:
why not raise it?

Reply:
Same as above that we are already exiting.

Reply:
And don't want to duplicate it.